README_cn.md

```diff
@@ -0,0 +1,422 @@
+| `"enabled"`                      | <p> 默认情况下录制功能是被禁用的。你可以通过修改`config.json`然后运行`/replay reload`来开启它，或者运行`/replay [enable\|disable]`指令。</p>                                                                                                                                                                                                                                                                                                                                                                  |
+| `"world_name"`                   | <p> 在录制文件中呈现的世界名称。 </p>                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| `"server_name"`                  | <p> 在录制文件中呈现的服务端名称。 </p>                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| `"player_recording_path"`        | <p> 玩家录制的保存路径。 </p>                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| `"chunk_recording_path"`         | <p> 区块录制的保存路径。 </p>                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| `"max_file_size"`                | <p> 回放文件允许录制的最大文件大小，这应当是一个数字+单位，例如 `5.2mb`。 </p> <p> 如果录制达到了这个限制，录制器将会停止。This is only approximate, expect the real file size to be slightly larger. 若要不对其进行限制，将此选项设置为 `0` 。 </p> <p> Be warned that this may impact server performance if your max file size is large, in order to check whether a file is too big (`>5GB`) it must be compressed which can be very expensive. You may check the time until the next file size check by running `/replay status`. </p>  |
```

Comment:
I have updated this description on the main branch, please update accordingly.

Reply:
“in order to check whether a file is too big (`>5GB`) it must be compressed which can be very expensive.” Is that means when checking a large file, it may consume much resources of server because of compressing the file?